docs/UserGuide.md

```diff
@@ -803,38 +885,75 @@ Command Options:
 | Any of the above | Min. one option | At least one option must be specified for the `find` command |
 
 Important notes:
-1. The `find` command word is case-insensitive, thus `FIND` is
-   considered an equivalent command word.
-
-2. *While each option is considered Optional, at least one option must be specified in total
 
+1. While all options are optional, at least one option must be specified in total.
+2. Only one instance of each option can be specified, i.e. `/n Friends n/Dinner` is invalid as the name
+      option is specified more than once.
 3. All keywords specified must match in order for a transaction to be displayed.
-4. Only one instance of each option can be specified, i.e. `/n Friends n/Dinner` is invalid as the name 
-option is specified more than once.
-5. For each keyword, a full substring match is required, thus `find n/Lunch with friends` with search for transactions
-whose name contains the String `Lunch with friends` and not `Lunch`, `with`, and `friends` separately. However, an 
-exact match is not required as a transaction with the name `Lunch with friends outside` contains the above substring
-and therefore it will be flagged as a match.
+4. For each keyword, a substring match is required, thus `find n/with friends` will search for transactions
+whose name contains the string "with friends". However, an exact word match is not required
+thus, a transaction with the name `Lunch with friends outside` will be flagged as a match.
+
+<div class="callout callout-important" markdown="span" style="margin-bottom: 20px;">
+Using the `find` command modifies the current `Transactions List` configuration, until
+another configuration modifying command (like `get_total_expenditure` or `list`) is used. Thus,
+editing transactions with the `edit` command will cause those transactions to be removed from
+the current `Transactions List` configuration if the properties being edited are the same
+properties used to filter the `Transactions List`. Thus, if the newly edited transaction do
+not match the conditions of the current `Transactions List` configuration, they will be removed
+from display in the current `Transactions List`.
```

Comment:
I think the explanation could be reduced to something like:

The `find` command, like `get_total_expenditure`, creates a filter on the transactions list and this filter persists across commands. 

Therefore, it is expected that any changes to the transaction list after `find` may result in transactions being hidden as they may no longer abide by the filter applied `find`.

Use the `list` command when this occurs to view all transactions.

Reply:
I've added that, thanks.